package.json

```diff
@@ -49,32 +50,35 @@
   "license": "MIT",
   "dependencies": {
     "babel-runtime": "6.11.6",
-    "carto": "CartoDB/carto#master"
+    "carto": "CartoDB/carto#master",
```

Comment:
be careful when pointing to a non-tag because you could be generating a version that does not work without noticing